### DISCUSSION TITLE
posts/proxmox-helper-scripts/

### DISCUSSION THREAD

**Discussion by giscus** [techno-tim/techno-tim.github.io#328] [Comments]
# posts/proxmox-helper-scripts/

Proxmox helper scripts is a collection of scripts to help you easily make changes to your Proxmox VE server along with installing many LXC Containers. This makes installing, configuring, and maintaining your Proxmox server in your HomeLab along with many applications as simple as running a script.

https://technotim.live/posts/proxmox-helper-scripts/

**Comment by bedge117** [comment]
I love using Helper Scripts.  I would also love it if you would do a video on Technitium's DNS.  I have been using it, and I love it.  I have my own signed DNSSEC zone and everything.  

Get away from that pi-hole, and use Traefik for your reverse-proxy with SSL with Let's Encrypt!!  You can also designate a secondary Zone instead of messing with the keepalived.  

Hope to see this video soon.  :)
